uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/BatchIndex.java

```diff
@@ -105,56 +107,69 @@ public void run() {
         }
     }
 
+    private boolean hasContent(Path dir) {
+        // TODO remove this filter when AF-1073 is resolved
```

Comment:
Please add a note to that jira that this workaround has to be removed (when somebody gets to fixing that).

Reply:
Done.